docs/sources/structure/topic-types/concept/index.md

```diff
@@ -47,15 +48,15 @@ A _concept_ topic includes the following elements:
 
 To write a concept topic, follow these steps.
 
-1. Decide which top-level entity you want to add documentation to by reviewing Grafana Labs' product documentation.
+1. Decide which top-level entity you want to add documentation to by reviewing product documentation.
```

Comment:
Thinking of this from the standpoint of a new contributor...

* What's a "top level entity"?
* What product documentation? 

This step seems vague and I'm not sure what it's trying to tell me to do.

Reply:
I think I'm best to rope @chri2547 here because as far as I can tell, they wrote this originally (https://github.com/grafana/writers-toolkit/commit/422b9f7754c706097f256bc781bb492482534f33#diff-9ce7098583ee205cba11c4191a455b0729fb9588cac7424ca45708c3eb21a961R80-R106).

My interpretation is that this is asking you to identify the concept that you'd like to write about by looking through the product documentation to understand which concepts haven't already been explained. 

If that is the intent, I can update the wording to that effect.

Reply:
@chri2547 did you have any comment on this before I merge this review?

Reply:
@jdbaldry , I can see that being a bit confusing. We have a similar pattern with task and reference topics, but the wording is a bit less confusing (perhaps?). See https://grafana.com/docs/writers-toolkit/structure/topic-types/task/#write-a-task-topic where we say "Determine where you want to add task documentation to the Grafana Labs product documentation." 

On a related note, part of what's missing in the toolkit is an explanation of the docs IA in general and how to build out the directory structure in a repo. We address this in the docs 101 training that Brenda and I put together, but that content hasn't made its way to the toolkit. A few slides starting here: https://docs.google.com/presentation/d/1_d_GF1W_8wa21g2HpIe-K4lY74z42g4Vq--A9i9nkN4/edit#slide=id.g156475cb98f_1_12

Reply:
That task description makes a lot more sense.... easier to parse etc.

Reply:
```suggestion
1. Determine where you want to add concept documentation to the Grafana Labs product documentation.
```

Reply:
I've opened https://github.com/grafana/writers-toolkit/issues/711 to track the writing of a page explaining documentation information architecture in general